libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -178,16 +181,18 @@ class RSCGovFloatParam {
     static const struct AP_Param::GroupInfo var_info[];
 
     void set_gov_enable(int8_t setenable) {enable = setenable; }
-    int16_t get_setpoint() { return setpoint; }
+    int16_t get_reference() { return reference; }
+    float get_range() { return range; }
     float get_disengage() { return disengage; }
-    float get_droop_setting() { return droop_setting; }
+    float get_droop_response() { return droop_response; }
     float get_tc() { return tc; }
 
 private:
     AP_Int8   enable;
-    AP_Float  disengage;    // sets the throttle percent where the governor disengages for return to flight idle
-    AP_Int16  setpoint;     // sets rotor speed for governor
-    AP_Float  droop_setting;// governor response to droop under load
-    AP_Float  tc;           // governor throttle curve weighting, range 50-100%
+    AP_Int16  reference;      // sets rotor speed for governor
```

Comment:
"reference" could be "target_rpm"?

Reply:
On this one, the term "reference" was taken from Woodward's engineering whitepapers on droop governors. It is called a 'reference' because it is not a setpoint. The governor moves the reference, as needed, to compensate for load so it is constantly a moving target when the governor is operating.

See:http://www.woodward.com/WorkArea/DownloadAsset.aspx?id=2147483661

Reply:
On this one I'm not comfortable with changing the variable name due to the design of this type of governor. It truly is a _governor_reference and not a 'target_rpm'. If the governor reference is 1500 and it droops by 20 it will move the reference to 1520 by adding droop. A droop governor never actually achieves such a thing as a 'target_rpm'. It always operates in droop, with droop compensation by moving the reference.